include/account/address.h

```diff
@@ -11,7 +11,7 @@ class Address
 
     Address(bytes const &publicKey);
 
-    Address(std::string bech32Address);
+    Address(std::string const &bech32Address);
```

Comment:
Here's a [fun discussion](https://stackoverflow.com/questions/10231349/are-the-days-of-passing-const-stdstring-as-a-parameter-over): passing by value (as before) was all right (even preferred IMHO) in your case - your implementation of the constructor now calls the copy constructor of std::string anyway.

Reply:
My experience with newer C++ standards is pretty limited (I  used to work on embedded C mostly and never used standard libraries) so I used `const &` every time for memory reasons. Recently I've learnt what you mentioned: 

> to pass by value and use std::move(std::string). 

I think in [this](https://github.com/ElrondNetwork/elrond-sdk-erdcpp/pull/4/commits/bed531d55813d0614166a8c186f7388c1a09deaa) PR I took care of it. Will take care of this cases in the future, thx for pointing out !